lib/api/web-element/scoped-element.js

```diff
@@ -265,9 +265,7 @@ class ScopedWebElement {
       this.webElement = promise;
     } else {
       this.webElement = new WebElementPromise(this.driver, this.then(async (element) => {
-        await promise;
-
-        return element;
+        return await promise;
```

Comment:
This change would impact a lot of commands, essentially all the action commands like click, sendKeys, etc.

Reply:
fixed in [aa56804](https://github.com/nightwatchjs/nightwatch/pull/4261/commits/aa568047bc7ac71608074b9ea63fca847caac254)